release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -918,6 +918,20 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 
 * Previously, {product-title} master nodes were missing Ingress security group rules when they were schedulable and deployed on {rh-openstack}. As a result, {product-title} deployments on {rh-openstack} failed for compact clusters with no dedicated workers. This fix adds Ingress security group rules on {rh-openstack-first} when masters are schedulable. Now, you can deploy compact three-node clusters on {rh-openstack}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955544[*BZ#1955544*])
 
+* Previously, if you specified an invalid AWS region, the installation program continued to try and validate availability zones. This caused the installer to become unresponsive for 60 minutes before timing out. The installer now verifies the AWS region and service endpoints before availability zones, which reduces the amount of time the installer takes to report the error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2019977[*BZ#2019977*])
+
+* Previously, you could not install a cluster using VMware vSphere if the vCenter host name began with a number. The installer has been updated and no longer treats this type of host name as invalid. Now, a cluster deploys successfully when the vCenter host name begins with a number.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2021607[*BZ#2021607*])
+
+* Previously, if you specified a `custom disk instance` type when deploying a cluster on Microsoft Azure, the cluster might not deploy. This occurred because the installer incorrectly determined that the minimum resource requirements had been met. The installer has been updated, and now reports an error when the number of vcpus available for the instance type in the selected region does not meet the minimum resource requirements. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2025788[*BZ#2025788*])
```

Comment:
```suggestion
* Previously, if you specified a custom disk instance type when deploying a cluster on Microsoft Azure, the cluster might not deploy. This occurred because the installer incorrectly determined that the minimum resource requirements had been met. The installer has been updated, and now reports an error when the number of vcpus available for the instance type in the selected region does not meet the minimum resource requirements. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2025788[*BZ#2025788*])
```